### PR TITLE
change default vm sku to Standard_D2s_v3

### PR DESCRIPTION
**Pull request by xizhamsft**
Change the defaul vm sku to Standard_D2s_v3 for better disk performance
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S170

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -694,7 +694,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                windows_admin_username=None,
                windows_admin_password=None,
                kubernetes_version='',
-               node_vm_size="Standard_DS2_v2",
+               node_vm_size="Standard_D2s_v3",
```
Comment (yungezz) [inline src/aks-preview/azext_aks_preview/custom.py:697]: the change itself is ok. Is the default vm documented somewhere?

Comment (xizhamsft) [reply src/aks-preview/azext_aks_preview/custom.py:697]: @jluk , could you modify our documentation after the change is merged.

**Comment by xizhamsft** [comment]
@arrownj, the old default value works. Changing to the new one for better default performance.

**Pull request merged**